files/zuul-install-requirements-pip-master.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I would be for renaming this to `git-master` everywhere, but it is just my preference so you can ignore it.

Reply:
@rpitonak I also though about this, but finally everything is installed via pip, just ogr and sandcastle are installed via pip with exact link to github.  But I have no preferences here. So if you want I can change it

Reply:
+1 git master; pip master is very confusing

Reply:
renamed